auth0/structure_auth0_connection.go

```diff
@@ -919,6 +927,11 @@ func expandConnectionOptionsEmail(d ResourceData) (*management.ConnectionOptions
 		return nil, err
 	}
 
+	options.AuthParams = map[string]interface{}{}
+	if authParamsMap := Map(d, "auth_params"); authParamsMap != nil {
+		options.AuthParams = authParamsMap
+	}
+
```

Comment:
A `nil` value will get literally sent as `null` to Management API and populate the literal settings as such. Because this setting should always be a `map[string]string`, sending an empty map seems preferred over `null`.

Reply:
If AuthParams is nil it won't get marshalled into the request body, e.g. https://go.dev/play/p/8ZIaKIaQl2k